pages/test-evaluate/evaluation-tools/selecting-eval-tools.pl.md

```diff
@@ -0,0 +1,230 @@
+  </tr>
+  <tr>
+    <td>Trzeba pamiętać, że w niektórych przypadkach narzędzia mogą dawać nieprawidłowe wyniki.</td>
+    <td>Lupa powiększająca znak ostrzegawczy z wykrzyknikiem. Wyświetlanych jest wiele lup. </td>
+  </tr>
+  <tr>
+    <td>Dlatego nie polegaj nadmiernie na tym, co mówią narzędzia w stosunku do tego, jakie są rzeczywiste doświadczenia osób korzystających z Twojej witryny.</td>
+    <td>Na ekranie 12 pól, w każdym z nich inna osoba przed komputerem.</td>
+  </tr>
+  <tr>
+    <td>„Wybór narzędzi oceny dostępności cyfrowej” wyjaśnia, co narzędzia mogą, a czego nie mogą zrobić, oraz na co zwrócić uwagę, aby znaleźć narzędzia spełniające Twoje potrzeby.</td>
+    <td>Wybór narzędzi oceny dostępności cyfrowej. Pojawia się ikona zestawu narzędzi, następnie drogowskaz i ikona lornetki.</td>
+  </tr>
+  <tr>
+    <td>„Lista narzędzi oceny dostępności cyfrowej” zawiera filtry, które pomogą Ci znaleźć odpowiednie narzędzie dla Twojej konkretnej sytuacji.</td>
+    <td>Lista dokumentów z rozwijanym polem wyboru i ikoną filtra, pozostają tylko dwa dokumenty.</td>
+  </tr>
+  <tr>
+    <td>Z narzędziami i wiedzą, jak z nich prawidłowo korzystać, masz wszystko, czego potrzebujesz, aby skuteczniej znajdować problemy z dostępnością.</td>
+    <td>Osoba,przy niej ikony narzędzi i żarówki. Ikony łączą się ze sobą, tworząc szkło powiększające ze słowem „bariery”.</td>
+  </tr>
+  <tr>
+    <td>Dostępność internetowa: niezbędna dla niektórych, przydatna dla wszystkich.</td>
+    <td>Wokół komputera układają się ikony: ręki, oka, mózgu, ucha i ust z falą dźwiękową.</td>
+  </tr>
+  <tr>
+    <td>Aby uzyskać więcej informacji o narzędziach oceny dostępności, odwiedź stronę w3.o-r-g/W-A-I/evaluation.</td>
+    <td>Narzędzia oceny, logo W3C i Inicjatywy na Rzecz Dostępności Internetu (WAI).</td>
+  </tr>
+</tbody>
+</table>
+
+{% include excol.html type="end" %}
+
+## Co mogą zrobić narzędzia oceny, a *czego nie potrafią*
+{:#cannot}
+
+Narzędzia oceny dostępności stron internetowych mogą pomóc w szybkim rozpoznaniu potencjalnych barier dostępności. Można z nich korzystać na wszystkich etapach procesu projektowania i tworzenia stron internetowych. Narzędzia mogą zapewniać w pełni zautomatyzowane testy i pomagać w ręcznej ocenie.
+
+Automatyczne narzędzia nie mogą sprawdzić wszystkich aspektów dostępności. Wymagany jest ludzki osąd. Czasami narzędzia oceny mogą dawać fałszywe lub wprowadzające w błąd wyniki. Narzędzia oceny dostępności stron internetowych nie mogą jednoznacznie ocenić dostępności, mogą jedynie w tym pomóc.
+
+
+
+## Własności narzędzi oceny
+{:#features}
+
+Narzędzia oceny dostępności cyfrowej są skierowane do różnych odbiorców. Korzystają z nich projektanci, programiści, nietechniczni autorzy treści, testerzy jakości, a czasem także użytkownicy. Narzędzia oferują różne możliwości i funkcje, które użytkowniy mogą porównać i ocenić, aby dobrać narzędzia pod kątem swoich konkretnych potrzeb.
```

Comment:
@zwiastunsw @viking-pl Is there a distinct word in Polish for "end-users" (vs "users" in general)? Just checking: maybe "użytkownicy" is the right one.

Reply:
Yes, in Polish, one encounters the translation of "end-user" to "użytkownik końcowy" but this is term used in economics (user who consumes a product or service provided to him). In computer science it is a loan translation.  In fact, it doesn't explain anything